src/dipdup/package.py

```diff
@@ -83,6 +83,9 @@ def __init__(self, root: Path, quiet: bool = False) -> None:
         self._types: dict[str, type[BaseModel]] = {}
         self._evm_abis = EvmAbiManager(self)
         self._cairo_abis = CairoAbiManager(self)
+    
+    def __repr__(self) -> str:
+        return f'<Class {self.__class__.__name__} (root={self.root}, evm_abis={self._evm_abis}, cairo_abis={self._cairo_abis})>'
```

Comment:
Why have you chosen these two attributes specifically? 🤔 Please, leave only the `root` one.